lib/al/Library/Screen/ScreenFunction.h

```diff
@@ -0,0 +1,107 @@
+#pragma once
+
+#include <al/Library/IUse/IUseCamera.h>
+#include <common/aglRenderBuffer.h>
+#include <basis/seadTypes.h>
+#include <container/seadPtrArray.h>
+#include <math/seadVector.h>
+#include <math/seadMatrix.h>
+#include <gfx/seadCamera.h>
+
+namespace al {
+
+class SceneCameraInfo;
+class ScreenCapture;
+
+class ScreenCaptureExecutor {
+public:
+    ScreenCaptureExecutor(int);
+    ~ScreenCaptureExecutor();
+
+    void createScreenCapture(int, int, int);
+    void draw(agl::DrawContext*, const agl::RenderBuffer*, int) const;
+    void tryCapture(agl::DrawContext*, const agl::RenderBuffer*, int);
+    void tryCaptureAndDraw(agl::DrawContext*, const agl::RenderBuffer*, int);
+
+    void requestCapture(bool, int);
+    void onDraw(int screenCaptureIndex);
+    void offDraw(int screenCaptureIndex);
+    void offDraw();
+
+    bool isDraw(int) const;
+
+    void* vtable;
```

Comment:
The class inherits `al::HioNode` which creates the vtable, so this can be removed once that is fixed